wallet/wallet.go

```diff
@@ -1887,6 +1887,36 @@ func (w *Wallet) MasterPubKey(account uint32) (*hdkeychain.ExtendedKey, error) {
 	return extKey, nil
 }
 
+// CoinTypeKey returns the BIP0044 coin type private key for the passed account.
+func (w *Wallet) CoinTypeKey() (*hdkeychain.ExtendedKey, error) {
```

Comment:
This method is not used anywhere, please remove it.

Reply:
done @jrick